File: fix4.py
import os
import re

def fix_generate_article():
    filepath = 'src/lib/news/ai/generate-article.ts'
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()

    content = re.sub(r'\bdepthCorrection\b', 'repairContext', content)
    
    with open(filepath, 'w', encoding='utf-8') as f:
        f.write(content)

fix_generate_article()

File: fix4.js
const fs = require('fs');
let content = fs.readFileSync('src/lib/news/ai/editorial-depth-quality.ts', 'utf8');

content = content.replace(
  /const key = normalizePara\(p\)\.slice\(0, 160\);/g,
  'const key = normalizePara(p).slice(0, 300);'
);

fs.writeFileSync('src/lib/news/ai/editorial-depth-quality.ts', content, 'utf8');
